ajax_request/request.js

```diff
@@ -0,0 +1,31 @@
+function AjaxSubmit(){
```

Comment:
Functions usually follow camelCase in most languages. Usually classes are in PascalCase. You may want to change this. More info: https://www.w3schools.com/js/js_conventions.asp